scopes/design/ui/cli-snippet/cli-snippet.compositions.tsx

```diff
@@ -12,3 +12,22 @@ export const CliSnippetExample = () => {
     </ThemeCompositions>
   );
 };
+
+export const CliSnippetLongExample = () => {
+  const log =
```

Comment:
this makes the code difficult to read, so please move out of composition and rename to be `MOCK_LOG` or similar